### PR TITLE
Federated ETL Backups and Alerting

### PR DESCRIPTION
**Pull request by thomasvn**
New doc providing recommendations for how to increase stability & recoverability when deploying in Federated ETL architecture

Builds upon this existing PR: https://github.com/kubecost/docs/pull/593

cc @jessegoodier

### CODE REVIEW

**Comment by jessegoodier** [comment]
Thank you for putting this together! 
Should we also link to?
Kubecost backups: [etl-backup](etl-backup.md).

**Comment by thomasvn** [comment]
@jessegoodier I didn't link the user to `etl-backup.md` because in a sense, the Federated ETL Architecture is already performing ETL backups by default. Thoughts?

**Review by jessegoodier — DISMISSED** [review]
I think this is good to go. I'll link to yours and remove redundancy from my PR.

File: federated-etl-backups-alerting.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Federated ETL Backups and Alerting
+
+This doc provides recommendations to improve the stability and recoverability of your Kubecost data when deploying in a Federated ETL architecture.
+
+  > **Note**: The Federated ETL Architecture is only supported on Kubecost Enterprise plans.
+
+## Option 1: Increase Prometheus retention
+
+Kubecost can rebuild its data (ETLs) using the Prometheus metrics from each cluster. It is strongly recommended to retain local cluster Prometheus metrics that meet an organization's disaster recovery requirements.
```
[the remark this answers is not in the document]
Comment (bstuder99) [reply federated-etl-backups-alerting.md:9]: I like that, can we also make sure ETL is spelt longhand in its first occurrence if I didn't already add that change?

[the remark this answers is not in the document]
Comment (thomasvn) [reply federated-etl-backups-alerting.md]: @bstuder99 Awesome, fixed!

**Comment by thomasvn** [comment]
Just resolved all comments. @bstuder99 or @jessegoodier , when possible could you approve again?

**Review by bstuder99 — APPROVED** [review]

**Pull request merged**